cli/flox-rust-sdk/src/models/environment/core_environment.rs

```diff
@@ -699,12 +699,22 @@ mod tests {
 
     #[cfg(feature = "impure-unit-tests")]
     use serial_test::serial;
+    use tempfile::TempDir;
 
     use super::*;
     use crate::flox::tests::flox_instance;
     #[cfg(feature = "impure-unit-tests")]
     use crate::models::environment::init_global_manifest;
 
+    fn empty_core_environment() -> (CoreEnvironment, Flox, TempDir) {
```

Comment:
Suggestion: document what empty means here

Reply:
Also should probably take a flox instance instead of creating a test instance itself

Reply:
Added a comment. For the current tests taking a flox instance would add more boilerplate, so I'd rather not do that, but we can change it if we need to use it for tests that work differently in the future